InfiniteBinarySeach.java

```diff
@@ -0,0 +1,54 @@
+// Time Complexity : O(log n)
```

Comment:
define n

Reply:
n is the index of the target in an array